packages/worker/src/api/controllers/global/tests/configs.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { configs } from "@budibase/backend-core"
+import { UserCtx } from "@budibase/types"
+import * as pro from "@budibase/pro"
+import { find, verifyAIConfig } from "../configs"
+
+jest.mock("@budibase/backend-core", () => ({
+  ...jest.requireActual("@budibase/backend-core"),
+  configs: {
+    getConfig: jest.fn(),
+    save: jest.fn(),
+  },
+}))
```

Comment:
Are these configs saved into CouchDB? Would it be difficult to _not_ mock them?

Reply:
They are saved in couch, but the main purpose of this test was to just test the mapping code for secrets and the default Budibase AI config. I didn't think it was necessary to have a DB save step for testing this controller

Reply:
We've generally been trying to test as much real code as possible, and avoiding mocks unless absolutely necessary. One of the motives is because it'll make any eventual move away from Jest easier, as other testing frameworks seem to do mocking differently.

It's not a huge deal, but if it's easy enough to actually do the DB save that would be preferable in the long term.